src/upgrader_canister/src/inspect_message.rs

```diff
@@ -0,0 +1,42 @@
+// required by the inspect_message macro
+#[allow(unused_imports)]
+use ic_exports::ic_cdk::{self, api};
+use ic_exports::ic_kit::ic;
+
+use crate::canister::UpgraderCanister;
+use crate::state::UpgraderCanisterState;
+
+/// NOTE: inspect is disabled for non-wasm targets because without it we are getting a weird compilation error
+/// in CI:
+/// > multiple definition of `canister_inspect_message'
+#[cfg(target_family = "wasm")]
```

Comment:
in the bridge sdk we've used export-api feature instead of target family, is it better?

Reply:
No, target_family is better in this case